pysph/examples/gas_dynamics/accuracy_test_2d.py

```diff
@@ -13,6 +13,8 @@
 
 from pysph.sph.scheme import GasDScheme, ADKEScheme, GSPHScheme, SchemeChooser
 from pysph.sph.wc.crksph import CRKSPHScheme
+from pysph.sph.gas_dynamics.psph.scheme import PSPHScheme
```

Comment:
This is simply too long an import.  Can you instead put a `psph.py` in `sph/gas_dynamics/` and put the scheme also right there.  I think another directory for each scheme is not warranted.

Reply:
From the PR I see that you have also not put a `sph/gas_dynamics/psph/__init__.py` which is not needed with this refactor.